go/vt/vtgate/planbuilder/testdata/tpch_cases.json

```diff
@@ -566,149 +566,208 @@
+                          "sum(volume) * count(*) as revenue",
+                          ":2 as supp_nation",
+                          ":3 as l_year",
+                          ":4 as o_custkey",
+                          ":5 as weight_string(supp_nation)",
+                          ":6 as weight_string(l_year)"
+                        ],
                         "Inputs": [
                           {
-                            "OperatorType": "Route",
-                            "Variant": "Scatter",
-                            "Keyspace": {
-                              "Name": "main",
-                              "Sharded": true
-                            },
-                            "FieldQuery": "select shipping.l_year, shipping.volume, shipping.l_suppkey, shipping.l_orderkey, weight_string(shipping.l_year) from (select extract(year from l_shipdate) as l_year, l_extendedprice * (1 - l_discount) as volume, l_suppkey as l_suppkey, l_orderkey as l_orderkey from lineitem where 1 != 1) as shipping where 1 != 1",
-                            "Query": "select shipping.l_year, shipping.volume, shipping.l_suppkey, shipping.l_orderkey, weight_string(shipping.l_year) from (select extract(year from l_shipdate) as l_year, l_extendedprice * (1 - l_discount) as volume, l_suppkey as l_suppkey, l_orderkey as l_orderkey from lineitem where l_shipdate between date('1995-01-01') and date('1996-12-31')) as shipping",
-                            "Table": "lineitem"
-                          },
-                          {
-                            "OperatorType": "Route",
-                            "Variant": "EqualUnique",
-                            "Keyspace": {
-                              "Name": "main",
-                              "Sharded": true
+                            "OperatorType": "Join",
+                            "Variant": "Join",
+                            "JoinColumnIndexes": "L:0,R:0,R:1,L:1,L:2,R:2,L:4",
+                            "JoinVars": {
+                              "l_suppkey": 3
                             },
-                            "FieldQuery": "select shipping.o_custkey from (select o_custkey as o_custkey from orders where 1 != 1) as shipping where 1 != 1",
-                            "Query": "select shipping.o_custkey from (select o_custkey as o_custkey from orders where o_orderkey = :l_orderkey) as shipping",
-                            "Table": "orders",
-                            "Values": [
-                              ":l_orderkey"
-                            ],
-                            "Vindex": "hash"
+                            "TableName": "lineitem_orders_supplier_nation",
+                            "Inputs": [
+                              {
+                                "OperatorType": "Projection",
+                                "Expressions": [
+                                  "sum(volume) * count(*) as revenue",
```

Comment:
can you explain why we are not passing `volume` as an index, like we do for the other columns?

Reply:
we are, the printing of the plan is just not showing it